server/db/index.js

```diff
@@ -0,0 +1,12 @@
+const { Pool } = require('pg');
+require('dotenv').config();
```

Comment:
this shouldn't be necessary; the first dotenv config (server/index.js) will instantiate variables to every running script in the process